CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ if(Arrow_FOUND)
   add_definitions(-D_ARROW_EXIST)
 endif()
 
-set(CMAKE_CXX_FLAGS "-std=gnu++17 -ggdb -Wnon-virtual-dtor -Wreorder -Wunused-variable -Wtype-limits -Wsign-compare -Wmaybe-uninitialized -lprofiler -O3")
+set(CMAKE_CXX_FLAGS "-std=gnu++17 -ggdb -Wnon-virtual-dtor -Wreorder -Wunused-variable -Wtype-limits -Wsign-compare -Wmaybe-uninitialized -lprofiler -O3 -pthread")
```

Comment:
adding -pthread , -lprofiler to CMAKE_CXX_FLAGS is not portable 
please review https://github.com/ceph/s3select/pull/98/

Reply:
done